src/app/user/services/user-store.service.ts

```diff
@@ -1,18 +1,46 @@
-import { Injectable } from "@angular/core";
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { UserService } from './user.service';
+
+export const ADMIN_EMAIL = 'admin@email.com';
```

Comment:
Exposing some credentials in your code is not safe. 🥹

Reply:
Yes it is eliminated and .role is checked from me api to decide if the user is admin